packages/dx-react-demos/src/bootstrap3/basic/table-cell-template.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Grid,
+  TableView,
+  TableHeaderRow,
+} from '@devexpress/dx-react-grid-bootstrap3';
+
+import {
+  generateRows,
+  globalSalesValues,
+} from '../../demo-data/generator';
+
+export default class Demo extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      columns: [
+        { name: 'region', title: 'Region' },
+        { name: 'sector', title: 'Sector' },
+        { name: 'channel', title: 'Channel' },
+        { name: 'customer', title: 'Customer' },
+        { name: 'product', title: 'Product' },
+        { name: 'amount', title: 'Sale Amount' },
+      ],
+      rows: generateRows({ columnValues: globalSalesValues, length: 14 }),
+    };
+  }
+  render() {
+    const { rows, columns } = this.state;
+
+    return (
+      <Grid
+        rows={rows}
+        columns={columns}
+      >
+        <TableView
+          tableCellTemplate={({ column, value, style, colSpan }) => (
```

Comment:
Please, do not inline templates. It forces Grid to render each time the parent component renders.

Reply:
Looks like we'll need the 'Performance Guide' soon )